docs/source/intel/optimization.mdx

```diff
@@ -0,0 +1,17 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Optimization
+
+## IncOptimizer
+
+[[autodoc]] intel.neural_compressor.optimizer.IncOptimizer
```

Comment:
For consistency with the `onnxruntime` package, I suggest we rename this module to `intel.neural_compressor.optimization`

Reply:
Yes, that's a good point, I was also thinking to rename `intel.neural_compressor.config` to `intel.neural_compressor.configuration` for the same reason. I agree that this is an important change that should be done as soon as possible (before our next release, which should be soon)

Reply:
OK great, let's tackle that in a separate PR :)